### DISCUSSION TITLE
'Yuzu Settings' creates a config file just Tears of the Kingdom, right?

### DISCUSSION THREAD

**Discussion by gaklimited** [MaxLastBreath/nx-optimizer#6] [Q&A] [answered]
<img width="193" alt="2023-07-30 (21h32m38s) TOTK MOD Manager 1 1 1  TOTK_MOD_Manager_1 1 1" src="https://github.com/MaxLastBreath/TOTK-mods/assets/66001816/23b8b4ea-6b20-4a5e-8ee6-3dcb20cfda05">

Or does it change Global settings? I can see that resolution changes the custom INI in the console, but I wanted to clarify for this one too.

**Comment by gaklimited** [comment, marked as answer]
Everything seems to go in 0100F2C0115B6000 folders of load and config.

**Comment by MaxLastBreath** [comment]
Yes everything goes into specifically for TOTK, no global settings are touched!
